dotnet/test/Microsoft.AutoGen.Integration.Tests/InMemoryRuntimeIntegrationTests.cs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// InMemoryRuntimeIntegrationTests.cs
+using Xunit.Abstractions;
+
+namespace Microsoft.AutoGen.Integration.Tests;
+
+public class InMemoryRuntimeIntegrationTests(ITestOutputHelper testOutput)
+{
+
+    [Theory, Trait("type", "integration")]
+    [MemberData(nameof(AppHostAssemblies))]
+    public async Task HelloAgentsE2EInMemory(string appHostAssemblyPath)
+    {
+        var appHost = await DistributedApplicationTestFactory.CreateAsync(appHostAssemblyPath, testOutput);
+        await using var app = await appHost.BuildAsync().WaitAsync(TimeSpan.FromSeconds(15));
+
+        await app.StartAsync().WaitAsync(TimeSpan.FromSeconds(120));
+        await app.WaitForResourcesAsync().WaitAsync(TimeSpan.FromSeconds(120));
+
+        await app.StartAsync().WaitAsync(TimeSpan.FromSeconds(120));
+        await app.WaitForResourcesAsync().WaitAsync(TimeSpan.FromSeconds(120));
+
+        //sleep 5 seconds to make sure the app is running
+        await Task.Delay(5000);
```

Comment:
Maybe extend it to 15000 seconds? 5s not enough for my windows PC to fully start the host

Reply:
done/ Let's get you a new PC!

Reply:
Already got a new PC this April LOL. Still take more than 5 seconds to see the agent logs